tests/unit-tests/api/customers.php

```diff
@@ -29,88 +29,90 @@ public function test_register_routes() {
 		$this->assertArrayHasKey( '/wc/v1/customers/batch', $routes );
 	}
 
-    /**
-     * Test getting customers.
-     *
-     * @since 2.7.0
-     */
-    public function test_get_customers() {
-	    wp_set_current_user( 1 );
+	/**
+	 * Test getting customers.
+	 *
+	 * @since 2.7.0
+	 */
+	public function test_get_customers() {
+		wp_set_current_user( 1 );
```

Comment:
I took that out in some of them - I don't know why, but that was breaking multisite unit tests. I couldn't figure out why the IDs/users were not matching up so I just used 1. The test suite always has an ID 1, and thats present on all network sites as superuser.